src/commands/build/build_project.py

```diff
@@ -9,10 +9,56 @@
 from starkware.starkware_utils.error_handling import StarkException
 
 from src.commands.build.build_exceptions import CairoCompilationException
+from src.cli.command import Command
 from src.utils.config.project import Project
 from src.utils.starknet_compilation import StarknetCompiler
 
 
+class BuildCommand(Command):
+    def __init__(self, project: Project) -> None:
+        super().__init__()
+        self._project = project
+
+    @property
+    def name(self) -> str:
+        return "build"
+
+    @property
+    def description(self) -> str:
+        return "Compile contracts."
+
+    @property
+    def example(self) -> Optional[str]:
+        return "$ protostar build"
+
+    @property
+    def arguments(self) -> List[Command.Argument]:
+        return [
+            Command.Argument(
```

Comment:
Can we do typed dicts instead of `Command.Argument`, and wrap it underneath?

Reply:
I'd rather avoid `TypedDict` because they make refactoring (renaming specifically) harder. Why do you want to use `TypedDict`?